docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -15,21 +15,32 @@ endif::[]
 
 // tag::how-remote-clusters-work[]
 Remote cluster connections work by configuring a remote cluster and connecting
-only to a limited number of nodes in that remote cluster. Each remote cluster
-is referenced by a name and a list of seed nodes. When a remote cluster is
-registered, its cluster state is retrieved from one of the seed nodes and up
-to three _gateway nodes_ are selected to be connected to as part of remote
-cluster requests.
-// end::how-remote-clusters-work[]
+only to a limited number of nodes in that remote cluster. There are two
+potential modes for remote cluster connections: sniff mode and proxy mode.
 
 All the communication required between different clusters
 goes through the <<modules-transport,transport layer>>. Remote cluster
 connections consist of uni-directional connections from the coordinating
-node to the selected remote _gateway nodes_ only.
+node to the remote remote connections.
+// end::how-remote-clusters-work[]
+
+[float]
+[[sniff-mode]]
+=== Sniff mode
+
+// tag::how-sniff-mode-works[]
```

Comment:
It doesn't look like this section is reused anywhere. I would remove the tags until the content is actually reused.
```suggestion
```